source/manage/logging.rst

```diff
@@ -0,0 +1,487 @@
+- `Output file log as plain text <https://docs.mattermost.com/configure/environment-configuration-settings.html#output-file-logs-as-json>`__
+
+.. note::
+
+    Changing the file log level to ``DEBUG`` also includes the request body of incoming webhooks in logs.
+
+You can optionally output log records to any combination of `console <#console-target-configuration-options>`__, `local file <#file-target-configuration-options>`__, `syslog <#syslog-target-configuration-options>`__, and `TCP socket <#tcp-target-configuration-options>`__ targets, each featuring additional customization. See `Advanced Logging <#advanced-logging>`__ for details.
+
+Define logging output
+---------------------
+
+.. code-block:: JSON
+
+    {
+        "console1": {
+            "Type": "console",
+            "Format": "json",
+            "Levels": [
+                {"ID": 5, "Name": "debug", "Stacktrace": false},
+                {"ID": 4, "Name": "info", "Stacktrace": false, "color": 36},
+                {"ID": 3, "Name": "warn", "Stacktrace": false},
+                {"ID": 2, "Name": "error", "Stacktrace": true, "color": 31},
+                {"ID": 1, "Name": "fatal", "Stacktrace": true, "color": 31},
+                {"ID": 0, "Name": "panic", "Stacktrace": true, "color": 31},
+                {"ID": 10, "Name": "stdlog", "Stacktrace": false}
+            ],
+            "Options": {
+                "Out": "stdout"
+            },
+            "MaxQueueSize": 1000
+        },
+        "file1": {
+            "Type": "file",
+            "Format": "json",
+            "Levels": [
+                {"ID": 5, "Name": "debug", "Stacktrace": false},
+                {"ID": 4, "Name": "info", "Stacktrace": false},
+                {"ID": 3, "Name": "warn", "Stacktrace": false},
+                {"ID": 2, "Name": "error", "Stacktrace": true},
+                {"ID": 1, "Name": "fatal", "Stacktrace": true},
+                {"ID": 0, "Name": "panic", "Stacktrace": true}
+            ],
+            "Options": {
+                "Compress": true,
+                "Filename": "mattermost_logging.log",
+                "MaxAgeDays": 1,
+                "MaxBackups": 10,
+                "MaxSizeMB": 100 
+            },
+            "MaxQueueSize": 1000
+        },
+        "file2": {
+            "Type": "file",
+            "Format": "json",
+            "Levels": [
+                {"ID": 2, "Name": "error", "Stacktrace": true},
+                {"ID": 1, "Name": "fatal", "Stacktrace": true},
+                {"ID": 0, "Name": "panic", "Stacktrace": true}
+            ],
+            "Options": {
+                "Compress": true,
+                "Filename": "mattermost_logging_errors.log", 
+                "MaxAgeDays": 30,
+                "MaxBackups": 10,
+                "MaxSizeMB": 100
+            },
+            "MaxQueueSize": 1000
+        }
+    }
+
+----
+
+Audit logging (Experimental|Beta)
```

Comment:
Also, how is this difference from the advanced logging section? The content seems roughly the same, and advanced logging even explains how to setup an audit log output

Reply:
I think maybe the specifier here is really just the different things you can do at each audit logs vs standard logs